src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -141,6 +171,28 @@ public void traverse(){
         for(ReferencePair pair : referencePairs){
             System.out.println(pair);
         }
+
+        if(referencePairs.size() != 1 && baseComparisonMode != BaseComparisonMode.NO_BASE_COMPARISON){
+            throw new UserException.BadInput("");
```

Comment:
Fill in the error message here. Eg., `"Base comparison mode " + baseComparisonMode + " only supports comparing two references at a time"`